.github/workflows/build-and-test.yml

```diff
@@ -2,9 +2,7 @@ name: Build
 
 on:
   push:
-    branches: [ master ]
   pull_request:
-    branches: [ master ]
```

Comment:
~~Removes `branches` [filter](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#using-filters) from `pull_request` event trigger.~~

~~https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request~~

> ~~Runs your workflow when activity on a pull request in the workflow's repository occurs. For example, if no activity types are specified, the workflow runs when a pull request is opened or reopened or when the head branch of the pull request is updated.~~

Reply:
~~This may be redundant if we're already running the workflow on every push.~~

Reply:
Addressed by https://github.com/muxinc/mux-stats-sdk-avplayer/pull/139/commits/8e486129422285ebb45fb7c452ce6381fed6f1d2 which removes the `pull_request` event trigger altogether.